cli/src/commands/bonds.rs

```diff
@@ -0,0 +1,57 @@
+use crate::commands::common::CommonCollectOptions;
+use crate::{dto::ValidatorBondRecord, utils::rpc::get_rpc_client};
+use anchor_lang::AccountDeserialize;
+use serde_yaml;
+use solana_account_decoder::UiAccountEncoding;
+use solana_client::{
+    rpc_config::{RpcAccountInfoConfig, RpcProgramAccountsConfig},
+    rpc_filter::Memcmp,
+};
+use validator_bonds::state::bond::Bond;
+
+const BONDS_DISCRIMINATOR: [u8; 8] = [224, 128, 48, 251, 182, 246, 111, 196];
```

Comment:
The bonds discriminator could be taken from dependency code and not needed to be hardcoded.
I believe it should be possible to do like

```
use validator_bonds::state::bond::Bond;
use use anchor_lang::Discriminator;

// use in code
let d = Bond::DISCRIMINATOR;
```